Pipfile

```diff
@@ -18,6 +18,11 @@ matplotlib = "*"
 pytest = "*"
 pytest-timeout = "*"
 pytest-cov = "*"
+hypothesis-auto = "*"
+hypothesis = "*"
```

Comment:
What is the use case of hypothesis module?

Reply:
https://hypothesis.readthedocs.io/en/latest/